lib/usage.js

```diff
@@ -151,23 +152,23 @@ module.exports = function (yargs) {
     }, {})
 
     if (keys.length) {
-      ui.div('Options:')
+      ui.div(__('Options:'))
 
       keys.forEach(function (key) {
         var kswitch = switches[key]
         var desc = descriptions[key] || ''
         var type = null
 
-        if (~options.boolean.indexOf(key)) type = '[boolean]'
-        if (~options.count.indexOf(key)) type = '[count]'
-        if (~options.string.indexOf(key)) type = '[string]'
-        if (~options.normalize.indexOf(key)) type = '[string]'
-        if (~options.array.indexOf(key)) type = '[array]'
+        if (~options.boolean.indexOf(key)) type = '[' + __('boolean') + ']'
+        if (~options.count.indexOf(key)) type = '[' + __('count') + ']'
+        if (~options.string.indexOf(key)) type = '[' + __('string') + ']'
+        if (~options.normalize.indexOf(key)) type = '[' + __('string') + ']'
+        if (~options.array.indexOf(key)) type = '[' + __('array') + ']'
 
         var extra = [
             type,
-            demanded[key] ? '[required]' : null,
-            options.choices && options.choices[key] ? '[choices: ' +
+            demanded[key] ? '[' + __('required') + ']' : null,
+            options.choices && options.choices[key] ? '[' + __('choices') + ': ' +
```

Comment:
Most usage and validation text containing colons includes the colon as part of the localized string value (e.g. `__('Commands:')`, `__('Options:')`, `__('Examples:')`, etc), which seems appropriate. The only exceptions are here (`__('choices') + ': '`) and line 293 (`__('default') + ': '`). Perhaps the colon should be included with these as well?